src/form-layout.scss

```diff
@@ -0,0 +1,139 @@
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+
+/*!
+.fd-form-horizontal
+*/
+
+$block: #{$fd-namespace}-form-layout;
+
+.#{$block} {
+  @include fd-reset();
+
+  background-color: transparent;
+  background-color: var(--sapGroup_ContentBackground);
```

Comment:
In the specs, the background is `transparent`. Why are you setting it first to `transparent` and then to `var(--sapGroup_ContentBackground)`?